core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1385,10 +1385,22 @@ object KafkaConfig {
     }
     if (maybeSensitive) Password.HIDDEN else value
   }
+
+  def populateSynonyms(input: util.Map[_, _]): util.Map[Any, Any] = {
```

Comment:
A comment or docstring would be useful here explaining the motivation for this method

Reply:
Added.